docs/index.rst

```diff
@@ -32,6 +32,7 @@
    /rst/statistics_backend/statistics_backend
    /rst/listeners/listeners_index
    /rst/types/types_index
+   /rst/types/json_tags
```

Comment:
Right now `JSON Tags` is a new section in the documentation and not a subsection within `Types`. Being the header file within the `types` folder maybe it makes sense to keep it within the `Types` subsection.

Reply:
:+1: